pyproject.toml

```diff
@@ -6,16 +6,34 @@ requires = ["poetry-core"]
 known_third_party = ["locust"]
 profile = "black"
 
+[tool.mypy]
+files = ["."]
+
 [tool.poetry]
 authors = []
 description = "Basic load testing for Palace API"
 name = "palace-api-load-test"
+packages = [
+    { include = "circulation_load_test", from = "src" },
+]
 readme = "README.md"
 version = "0"
 
 [tool.poetry.dependencies]
+atoma = ">=0.0.17,<0.0.18"
 locust = "^2.12.1"
 python = ">=3.7,<4"
 
 [tool.poetry.group.dev.dependencies]
 pre-commit = "^2.20.0"
```

Comment:
It looks like we need tox and pytest to run the tests, would be good to add this to the `dev.dependencies` section, and add tox to the `ci.dependencies` section. Something like this: 
https://github.com/ThePalaceProject/circulation/blob/dc7132ac8ebb6371317ed47a420694acdb7a19f9/pyproject.toml#L234-L265